flake.nix

```diff
@@ -33,7 +33,7 @@
         inherit (haskell-nix) haskellLib;
 
         defaultGHCVersion = "ghc925";
-        ghcVersions = [ "ghc902" defaultGHCVersion ];
+        ghcVersions = [ "ghc902" defaultGHCVersion "ghc944" ];
```

Comment:
A minor nit: I think it would be actually better to write `"ghc925"` here instead of `defaultGHCVersion`. The reason is that if we change `defaultGHCVersion` to be e.g. `"ghc944"` we need to be careful to remember to update `ghcVersions` accordingly because it could as well become `["ghc902" "ghc944" "ghc944"]`, in other words, the way it is written one cannot just update `defaultGHCVersion` and be done, one has to remember more context about how it is used in order to keep things correct.

Reply:
Good point, what about
```nix
ghcVersions = [ "ghc925" "ghc902" "ghc944" ];
defaultGHCVersion = builtins.head ghcVersions;
```
?

Reply:
This sounds good to me!

Reply:
https://github.com/tweag/ormolu/compare/b0fdd9b08d69aebe67bbc945f827af6c32ee01fd..0d55b81c4656024dceb5805e51b62cf94d33c84a